### PR TITLE
Upgraded springboot version & completed migration of existing features

### PR DESCRIPTION
**Pull request by xcapdevila**
This PR completes the migration of the existing features. (Related to issue #19 )

### CODE REVIEW

**Comment by maxgrabenhorst** [comment]
Hi @xcapdevila,
thanks so much for this PR!! This really helps me to focus on the core development of messenger4j!
The next 3 weeks I'm very very busy so I will not have the time to review it but afterwards it's the first thing I will work on. It's really valuable to finally merge the rewritten quick-start-template to master to make it available to everyone.

File: src/main/java/com/github/messenger4j/quickstart/boot/MessengerPlatformCallbackHandler.java [hunk 1]
```diff
@@ -73,508 +64,437 @@
...
+
+		final TemplateMessage templateMessage = TemplateMessage.create(buttonTemplate);
+		final MessagePayload messagePayload = MessagePayload.create(recipientId, templateMessage);
+		this.messenger.send(messagePayload);
+	}
+
+	private void handleAttachmentMessageEvent(AttachmentMessageEvent event) {
+		logger.debug("Handling QuickReplyMessageEvent");
+		for (Attachment attachment : event.attachments()) {
+			final String senderId = event.senderId();
```
Comment (Plotron) [inline src/main/java/com/github/messenger4j/quickstart/boot/MessengerPlatformCallbackHandler.java]: I think this should be outside of the loop.

Comment (xcapdevila) [reply src/main/java/com/github/messenger4j/quickstart/boot/MessengerPlatformCallbackHandler.java]: You are right. Solved.

**Comment by maxgrabenhorst** [comment]
Thank you so much @xcapdevila for your great effort to migrate this project! Much appreciated!
Thanks @Plotron for reviewing!

**Pull request merged**